core/domain/activity_domain.py

```diff
@@ -27,8 +27,8 @@ class ActivityReference(object):
     collections.
 
     Attributes:
-        activity_type: A string indicating the activity type.
-        activity_id: A string indicating the activity id.
+        type: A string denoting type of activity.
```

Comment:
Actually, let's use this format, so that it's consistent with some of our other services docstrings:
```
        type: str. The activity type.
        id: str. The activity id.
```